roles/addons/report/tasks/main.yml

```diff
@@ -81,10 +61,8 @@
   tags:
     - template
   when:
-    - report_settings.inventory_checker is defined
-    - report_settings.inventory_checker is not none
-    - report_settings.inventory_checker
-    - report_settings.inventory_checker is iterable
+    - report_inventory_checker is defined
```

Comment:
It's in the defaults now, it is always defined.

Reply:
Yes, I will remove that. Thanks for pointing that out.